src/components/expansion/ExpansionRuleForm.svelte

```diff
@@ -211,7 +211,7 @@
 <PageTitle title={pageTitle} />
 
 <CssGrid bind:columns={$expansionRulesFormColumns}>
-  <Panel overflowYBody="hidden" padBody={false}>
+  <Panel padBody={false}>
```

Comment:
Does hiding the Y-overflow not matter anymore here?

Reply:
We actually want the panel body to overflow - previously it was hiding Y overflow and if your window was too small the form would not scroll. Was there a reason to have it hidden before?

Reply:
Sounds good, it was just not needed before